Update BEK Support

EricBergDE opened the pull request:
Since Dec 2017 BEK Backup is fully supported. https://azure.microsoft.com/en-us/blog/azure-backup-now-supports-bek-encrypted-azure-virtual-machines/

azurecla commented:
Hi __@GaugeField__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (abhic). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

EricBergDE committed:
Update BEK Support

Since Dec 2017 BEK Backup is fully supported. https://azure.microsoft.com/en-us/blog/azure-backup-now-supports-bek-encrypted-azure-virtual-machines/
